app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class VotesController < ApplicationController
+
+  def create
+    current_user.votes.create(votable_type: vote_params[0], votable_id: vote_params[1])
```

Comment:
Zach, in regard to (2), ran into an issue attempting to do that. I want both parameters to be required but params.require only accepts a single argument. The array input works, but vote_params is produced as an array as well instead of a hash. With that in mind, any thoughts on how to make it cleaner? Thanks!